Description: Adds a missing manpage
 gts2xyz does not include a manpage. 
Author: Ruben Molina <rmolina@udea.edu.co>
Last-Update: 2013-09-04

--- /dev/null
+++ gts-0.7.6+darcs121130/doc/manpages/gts2xyz.1
@@ -0,0 +1,19 @@
+.TH gts2xyz 1 "September 4, 2013" "" "GNU Triangulated Surface utils"
+
+.SH NAME
+gts2xyz \- samples a GTS file on a regular grid at intervals DX. 
+
+.SH SYNOPSIS
+.B gts2xyz
+.RI DX <\  input.gts\  >\  output.xyz
+
+.SH DESCRIPTION
+This manual page documents briefly the 
+.B gts2xyz
+command.
+
+.SH AUTHOR
+gts2xyz was written by Stephane Popinet <popinet@users.sourceforge.net>.
+.PP
+This manual page was written by Ruben Molina <rmolina@udea.edu.co>,
+for the Debian project (but may be used by others).
